js/mocha-tests/test/ES6Class.test.js

```diff
@@ -0,0 +1,24 @@
+// use named imports
+import { ES6Class } from '../src/ES6Class';
+
+// Use sinon to spy and stub
+describe('sinon', () => {
+  before(() => {
```

Comment:
If I understand this correctly, sinon will stub the method on the prototype of `ES6Class`. Would you consider stubbing an instance instead? This changes the behavior of every instance of `ES6Class` instead of the instance you're testing below.